tutorials/image-curation/image-curation.ipynb

```diff
@@ -0,0 +1,528 @@
+    "    --input_format \"parquet\" \\\n",
+    "    --output_folder mscoco \\\n",
+    "    --output_format webdataset \\\n",
+    "    --url_col \"URL\" \\\n",
+    "    --caption_col \"TEXT\" \\\n",
+    "    --processes_count 16 \\\n",
+    "    --thread_count 64 \\\n",
+    "    --resize_mode no"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os._exit(0) # Shut down all kernels to prevent dependencies from conflicting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install NeMo Curator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!pip install cython ipywidgets\n",
+    "# Install from source by default\n",
+    "!pip install --extra-index-url https://pypi.nvidia.com ../../[image]\n",
+    "# !pip install --extra-index-url https://pypi.nvidia.com nemo-curator[image]\n",
+    "%env DASK_DATAFRAME__QUERY_PLANNING False"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create CLIP Embeddings\n",
+    "\n",
+    "### Load the Dataset\n",
+    "Instead of operating on the images directly, most features in NeMo Curator take embeddings as inputs. So, as the first stage in the pipeline, we are going to generate embeddings for all the images in the dataset. To begin, let's load the dataset using NeMo Curator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo_curator import get_client\n",
+    "\n",
+    "client = get_client(cluster_type=\"gpu\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change the dataset path if you have your own dataset\n",
+    "dataset_path = \"./mscoco\"\n",
+    "# Change the unique identifier depending on your dataset\n",
+    "id_col = \"key\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo_curator.datasets import ImageTextPairDataset\n",
+    "\n",
+    "dataset = ImageTextPairDataset.from_webdataset(dataset_path, id_col)\n",
+    "# Filter out any entries that failed to download\n",
+    "dataset.metadata = dataset.metadata[dataset.metadata[\"error_message\"].isna()]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Choose the Embedder\n",
+    "We can now define the embedding creation step in our pipeline. NeMo Curator has support for all [timm](https://pypi.org/project/timm/) models. NeMo Curator's aesthetic classifier is trained on embeddings from `vit_large_patch14_clip_quickgelu_224.openai`, so we will use that.\n",
```

Comment:
Maybe add a link for more information about `vit_large_patch14_clip_quickgelu_224.openai`?